<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CS100 National Bank App</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>741</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>161</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/HP/.designer/backup/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>440</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>330</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
     </widget>
     <widget class="QPushButton" name="createAccountButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>510</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Create new account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>510</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="usernam_Label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>94</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fpasswordButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>560</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Forgot your password?</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome_Label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome back to your banking app!</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_Label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>89</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_fullname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Good evening, Shuja Abbas!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>sendMoney.png</normaloff>sendMoney.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit Money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>View Card Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>420</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Account Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>510</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>510</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Pay Bill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>View Transactio  History</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>510</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>420</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Budget Manager</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>420</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Currency Converter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Banking Features</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>311</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transaction History</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Balance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_balance">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>200,000 PKR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_accounttype">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Savings Account</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cardtype">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MasterCard Debit Card</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_accountnumber">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Account # 2100564656544654</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
